docs/topics/mta-rn-new-features-6-2-0.adoc

```diff
@@ -9,12 +9,12 @@
 
 This section describes the new features of the Migration Toolkit for Applications (MTA) 6.2.0.
 
-.Integration with JIRA
+.xref:../../../docs/web-console-guide/master.adoc#creating-configuring-jira-connection[Integration with JIRA]
```

Comment:
Why are we editing MTA new features in MTR release notes PR . These changes are un related . 
Can we put them in separate PR?